LICENSE.md

```diff
@@ -0,0 +1,21 @@
+## The MIT License (MIT)
+
```

Comment:
Because the MIT license is more popular, at least in the Node.js community, which makes it easy for people to adopt this project.